source/Applications/Advanced/Downsample/Downsample.cpp

```diff
@@ -175,23 +31,23 @@ int main()
     {
         Zivid::Application zivid;
 
-        std::string fileName = std::string(ZIVID_SAMPLE_DATA_DIR) + "/Zivid3D.zdf";
-        std::cout << "Reading ZDF frame from file: " << fileName << std::endl;
-        const auto frame = Zivid::Frame(fileName);
-
-        const auto pointCloud = frame.pointCloud();
-        std::cout << "Loaded " << pointCloud.size() << " data points" << std::endl;
+        std::string dataFile = std::string(ZIVID_SAMPLE_DATA_DIR) + "/Zivid3D.zdf";
+        std::cout << "Reading ZDF frame from file: " << dataFile << std::endl;
+        const auto frame = Zivid::Frame(dataFile);
 
-        auto downsamplingFactor = 4;
-        auto pointCloudDownsampled = downsample(pointCloud, downsamplingFactor);
+        auto pointCloud = frame.pointCloud();
+        std::cout << "Size of point cloud before downsampling: " << pointCloud.size() << " data points" << std::endl;
```

Comment:
Nitpick:
The output of this sample is now:
```
1. Size of point cloud before downsampling: 2861568 data points
2. Visualizing point cloud before downsampling
3. Setting up visualization
4. Visualizing point cloud
5. Running visualizer. Blocking until window closes
6. Downsampling point cloud
7. Size of point cloud after downsampling: 178848 data points
8. Visualizing point cloud after downsampling
9. Setting up visualization
10. Visualizing point cloud
11. Running visualizer. Blocking until window closes
```
**Line 2-5 and 8-11 is basically the same information given four lines in a row.
I think you can remove two of the printouts from inside `visualizePointCloud()`,** given that is does not give any inconsistency with the other samples.